src/js/player.js

```diff
@@ -1371,11 +1371,6 @@ class MediaElementPlayer {
 
 			t.container.style.minWidth = `${siblingsWidth}px`;
 
-
```

Comment:
Condition this with something like the approach described on http://johanronsse.be/2016/01/03/simple-flexbox-check/; then it could append a class to `container` and then reintegrate the CSS you removed but targeting this new clase. Just a thought